wallet/src/Pos/Wallet/Web/Tracking/BListener.hs

```diff
@@ -86,6 +94,7 @@ onApplyBlocksWebWallet blunds = setLogger . reportTimeouts "apply" $ do
     -- something a bit more reasonable.
     pure mempty
   where
+    -- TODO(adn): Revisit this bit.
```

Comment:
Maybe elaborate on this if it's going to remain past your private branch. What needs revisiting?